src/components/SelectCoinModal.svelte

```diff
@@ -16,6 +16,17 @@
     dispatch('selected', { coin });
     active = false;
   };
+
+  // Show a loading message while we wait for tdex markets
+  // After a timeout delay, show a different message
+  const messages = {
```

Comment:
Juts personal preference, I do use `enum` as a more semantically correct for what they are
```suggestion
  enum MESSAGE = {
```